vendor/sigs.k8s.io/cluster-api/pkg/controller/machine/controller.go

```diff
@@ -211,6 +211,7 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 				return reconcile.Result{Requeue: true, RequeueAfter: requeueErr.RequeueAfter}, nil
 			}
 
+			klog.Errorf("Actuator returned error: %v", err)
```

Comment:
what is this change?

Reply:
oh awkward, made in error. I will need to open this change upstream, otherwise errors returned by Update() will not log.

Reply:
this is removed